src/utils/__tests__/TransactionResultUtil.test.ts

```diff
@@ -148,64 +143,7 @@ describe("TransactionResultUtil", () => {
     // Check EXPIRED cases !wasAuthRequested !sendPaymentResultOK
     expect(
       getViewOutcomeFromEcommerceResultCode(TransactionStatusEnum.EXPIRED)
-    ).toEqual(ViewOutcomeEnum.TIMEOUT);
-
-    // Check EXPIRED cases wasAuthRequested !sendPaymentResultOK (undefined)
-    expect(
```

Comment:
How come you deleted these unit tests ?

Reply:
This test has been deleted because we decided on the way to map all expired status to generic error. See RFC https://pagopa.atlassian.net/wiki/spaces/I/pages/703857491/Mapping+stato+transazione+-+pagina+di+esito

Reply:
Thank you for your response 🚀